base/interactiveutil.jl

```diff
@@ -676,13 +679,13 @@ end
 
 
 """
-    whos(io::IO=STDOUT, m::Module=current_module(), pattern::Regex=r"")
+    whos(io::IO=STDOUT, m::Module=Main, pattern::Regex=r"")
 
 Print information about exported global variables in a module, optionally restricted to those matching `pattern`.
 
 The memory consumption estimate is an approximate lower bound on the size of the internal structure of the object.
 """
-function whos(io::IO=STDOUT, m::Module=current_module(), pattern::Regex=r"")
+function whos(io::IO=STDOUT, m::Module=Main, pattern::Regex=r"")
```

Comment:
It's not actually a behavioral change in that case (both the old and new function would have gotten the argument value `Main` in either case). So this is basically just a bugfix that's being forced by the deprecation of this function. This confusion is basically why moving to `@__MODULE__` is a much simpler, more understandable API. (It is a behavioral change if you were calling this while defining a new module, even if called from the Main module, but we didn't have the `STDOUT` variable available then, so you wouldn't have been able to call this method anyways).